modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/custom_fields.js

```diff
@@ -1289,19 +1320,35 @@ AUI.add(
 
 										var domEvent = event.domEvent;
 
+										keysPressed[domEvent.keyCode] = true;
```

Comment:
[`keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

Reply:
I used that as we have already been using it, what should we use instead?

Reply:
Oh, you're right. I think this is actually an AUIEvent and not a native dom event. In that case, yeah lets just keep using keyCode since that is the preferred AUI way. 

Sorry for the sidetrack, this should be fine then

Reply:
okay, I thought so, I changed back.